app/components/TaskTemplateTile.tsx

```diff
@@ -0,0 +1,48 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { FunctionComponent } from 'react'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+
+type TaskTemplateTileTranslation = {
+  subtask: string,
+  edit: string
+}
+
+const defaultTaskTemplateTileTranslations: Record<Languages, TaskTemplateTileTranslation> = {
+  en: {
+    subtask: 'Subtasks',
+    edit: 'Edit'
+  },
+  de: {
+    subtask: 'Unteraufgabe',
+    edit: 'Bearbeiten'
+  }
+}
+
+export type TaskTemplateTileProps = {
+  templateName: string,
+  subtaskCount: number,
+  isSelected: boolean,
+  editClickedCallback: () => void,
+  tileClickedCallback: () => void
+}
+
+const TaskTemplateTile: FunctionComponent<PropsWithLanguage<TaskTemplateTileTranslation, TaskTemplateTileProps>> =
```

Comment:
`export const TaskTemplateTile = ...` and remove the `export { TaskTemplateTile }` at the end of the file